types/mailparser/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "dtslint/dt.json"
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-outside-dependencies": false
+    }
```

Comment:
```suggestion
    "extends": "dtslint/dt.json"
```

Reply:
CI will pass as soon 'iconv-lite' is white listed

Reply:
> CI will pass as soon 'iconv-lite' is white-listed

Sorry. I did not wait long enough after the tools were updated for the changes to actually take effect.  `tslint.json` has been removed from the PR.